stdlib/public/core/FloatingPoint.swift

```diff
@@ -1653,7 +1654,7 @@ extension FloatingPoint {
   ///
   /// - Parameter rule: The rounding rule to use.
   /// - Returns: The integral value found by rounding using `rule`.
-  @_transparent
+  @inline(__always)
```

Comment:
This should almost surely stay transparent, as we want them to a single inline instruction in all circumstances on x86 or arm64.

Reply:
Will it not with just `@inline(__always)`?

Reply:
If so, what does @_transparent mean, and why do we use it for arithmetic operators?

Reply:
I guess, this (👆) is a question for someone from @eeckstein's team.

Reply:
inline(__always) is a hint for the optimizer.
But transparent is also considered in Onone, mainly to get e.g. all the overflow diagnostics, which only work if arithmetic operations are inlined.
Also, transparent functions do not show up as function calls when debugging.

Reply:
Ah... The `Onone` argument is a good one. I did not think about that. Thanks!